packages/terra-signature/src/Signature.jsx

```diff
@@ -122,6 +137,15 @@ class Signature extends React.Component {
     }
   }
 
+  mouseOut(event) {
+    if (this.state.painting) {
+      this.setState({ painting: false });
+      if (this.props.onChange) {
+        this.props.onChange(event, this.state.lineSegments, this.state.lineSegments[this.state.lineSegments.length - 1]);
+      }
+    }
+  }
```

Comment:
Canceling the signature on mouse out adds a more crucial role of precision to the user and frequently cancels signatures midway. It would be worth exploring alternatives and getting opinions on how to handle this.

![signature](https://user-images.githubusercontent.com/6720991/31692763-d2502a42-b360-11e7-8bda-25123ba97fec.gif)

Reply:
Right, I talked to @dkasper-was-taken about this yesterday - the other alternative would be to add a timer, similar in hookshoot, where the signature can continue within a certain timeframe.

Reply:
Actually, this is a simpler solution - utilize `mouseleave` instead of `mouseout` ([differences](https://developer.mozilla.org/en-US/docs/Web/Events/mouseleave)) and attach that to the document.body. We can take advantage of `mouseleave` only being triggered by leaving the document window, so a signature can still be drawn when exiting the canvas element.